test/provision/compile.sh

```diff
@@ -45,13 +48,28 @@ then
 
       if [[ "${CILIUM_OPERATOR_IMAGE}" == "" ]]; then
         echo "building cilium-operator image..."
-        make LOCKDEBUG=1 docker-operator-image
-        echo "tagging cilium-operator image..."
+        make LOCKDEBUG=1 docker-operator-image docker-operator-aws-image docker-operator-azure-image docker-operator-generic-image
```

Comment:
We should probably reference https://github.com/cilium/cilium/issues/12035 here as well

(The equivalent change for the regular cilium build is in the makefile)